app/controllers/ops_controller/diagnostics.rb

```diff
@@ -350,6 +347,17 @@ def diagnostics_server_list
 
   private ############################
 
+  def fetch_journal_log(filter_params)
+    require "systemd/journal"
+    journal = Systemd::Journal.new
+    begin
+      journal.filter(filter_params)
+      journal.map(&:message)
+    ensure
+      journal.close
+    end
```

Comment:
Super minor, but is there a block form we can use?

```suggestion
    Systemd::Journal.new do |journal|
      journal.filter(filter_params)
      journal.map(&:message)
    end
```

Reply:
Oh potentially, let me check

Reply:
They have an `.open` method which yields and closes in an `ensure`